website/website/package.json

```diff
@@ -13,5 +13,8 @@
   },
   "dependencies": {
     "remarkable-admonitions": "^0.2.2"
+  },
+  "resolutions": {
+    "babel-plugin-dynamic-import-node": "2.3.0"
```

Comment:
Do we need this?

Reply:
Yes. There is a bug in some combinations of docusaurus and babel right now. I can find the thread I got this from if you want to read up on it.

Reply:
https://github.com/facebook/docusaurus/issues/2637